### ISSUE TITLE
Accessibility: Answer button lacks TalkBack label

### ISSUE THREAD

**Issue by mateuszmp** [signalapp/Signal-Android#11174]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
When calling a user with narration enabled (TalkBack) the buttons to Answer and Decline the call are unlabeled causing them to be read by the narration software as "unlabeled". This makes it almost impossible for visually impaired users to answer a Signal audio call.

### Steps to reproduce
- enable TalkBack in Android accessibility
- receive a call
- move finger onto the green answer button

**Actual result:** Narration says "unlabeled"
**Expected result:** Narration should say "Answer"

### Device info

**Device:** Motorola G6 Plus
**Android version:** 9
**Signal version:** 5.4.12
